.gitignore

```diff
@@ -13,26 +12,27 @@ tmp
 
 npm-debug.log*
 
-# Ignore everything in `dist` folder except for
-# the ESLint config & package.json files
+# Ignore built files in `dist` folder
+# Leave the package.json and wrappers
 /dist/*
 !/dist/package.json
-!/dist/jquery.bundler-require-wrapper.js
-!/dist/jquery.bundler-require-wrapper.slim.js
+!/dist/wrappers
 
-# Ignore everything in the `dist-module` folder except for the ESLint config,
-# package.json & Node module wrapper files
+# Ignore built files in `dist-module` folder
+# Leave the package.json and wrappers
 /dist-module/*
 !/dist-module/package.json
-!/dist-module/jquery.node-module-wrapper.js
-!/dist-module/jquery.node-module-wrapper.slim.js
+!/dist-module/wrappers
 
 /external
 /node_modules
 
 /test/data/core/jquery-iterability-transpiled.js
 /test/data/qunit-fixture.js
 
-# Ignore BrowserStack files
+# Release artifacts
+changelog.*
```

Comment:
Everything but changelogs go into the `tmp/` folder. Archives will be under `tmp/archives`. I had almost everything top-level originally, but that was a lot of `.gitignore` entries so I put everything under `tmp/` instead. The changelogs seemed a little different because `changelog.md` is copied to jquery-dist top-level and changelog.html is only for the blog post.

Reply:
Why not put `changelog.html` in `tmp` then?

Reply:
I'm not sure that's more helpful. You'd still need the extra `.gitignore` entry and `.*` is even shorter than `.md`.

Reply:
That said, I could just stick both in `tmp`.

Reply:
Actually, could we leave it? I was thinking we could commit the changelog.md to the tag as well, so it would need to stay top-level. And if the `.md` is top-level, it's a little more convenient for me to have the `.html` top-level. I can foresee myself continuing the post release, which removes everything before I've copied the HTML, but that's much less likely if I see the HTML top-level. Yes, I can always generate it again, but I like that it's more in my face.

Reply:
We discussed it on the meeting; we're leaving it as-is.